extensions/wikia/AdEngine/js/SlotTweaker.js

```diff
@@ -71,28 +69,6 @@ define('ext.wikia.adEngine.slotTweaker', [
 		}
 	}
 
-	// TODO: remove it after fully disabling Badge Ad: ADEN-6579
-	function adjustLeaderboardSize(slotname) {
-		var slot = doc.getElementById(slotname);
-		if (!geo.isProperGeo(instantGlobals.wgAdDriverDisableBadgeAdCountries) &&
-			isTopLeaderboard(slotname) && isStandardLeaderboardSize(slotname)) {
```

Comment:
`isStandardLeaderboardSize()` looks like it's not used now

Reply:
Yep, removed too.